### PR TITLE
Improve `rgh-feature-descriptions` links

### LINKED ISSUE #6600: `rgh-feature-descriptions` Add “Report bug” on feature pages
### Description

On our feature, we should add a simple new button that opens the bug report with the feature name in the title. 

### Example URLs

https://github.com/refined-github/refined-github/blob/main/source/features/align-issue-labels.css

### PR DESCRIPTION
**Pull request by fregante**
- Closes #6600 

#### Regular feature: https://github.com/refined-github/refined-github/blob/main/source/features/sync-pr-commit-title.tsx
Screenshot 11

#### CSS counterpart: https://github.com/refined-github/refined-github/blob/main/source/features/align-issue-labels.css
Screenshot 12

#### RGH feature: https://github.com/refined-github/refined-github/blob/main/source/features/rgh-feature-descriptions.css
Screenshot 13

#### CSS-only feature: https://github.com/refined-github/refined-github/blob/main/source/features/center-reactions-popup.css

Screenshot 14

### CODE REVIEW

File: source/features/rgh-feature-descriptions.tsx [hunk 1]
```diff
@@ -22,14 +22,18 @@ async function add(infoBanner: HTMLElement): Promise<void> {
 	}
 
 	const conversationsUrl = new URL('https://github.com/refined-github/refined-github/issues');
-	conversationsUrl.searchParams.set('q', `sort:updated-desc "${feature.id}"`);
+	conversationsUrl.searchParams.set('q', `sort:updated-desc is:open "${feature.id}"`);
```
Comment (fregante) [inline source/features/rgh-feature-descriptions.tsx:25]: Change: only show open issues/PRs by default. Seeing current bugs is more useful than "every discussion"

File: source/features/rgh-feature-descriptions.tsx [hunk 2]
```diff
@@ -22,14 +22,18 @@ async function add(infoBanner: HTMLElement): Promise<void> {
...
+
+	const newIssueUrl = new URL('https://github.com/refined-github/refined-github/issues/new');
+	newIssueUrl.searchParams.set('template', '1_bug_report.yml');
+	newIssueUrl.searchParams.set('title', `\`${feature.id}\`: `);
```
Comment (fregante) [inline source/features/rgh-feature-descriptions.tsx:29]: Example URL: https://github.com/refined-github/refined-github/issues/new?template=1_bug_report.yml&title=%60align-issue-labels%60%3A+

**Review by FloEdelmann — APPROVED** [review]
Nice :+1:

**Comment by fregante** [comment]
Extended the disabled feature banners

Screenshot 3

Screenshot 4

Screenshot 5

**Pull request merged**